includes/class-gp-translation-helpers.php

```diff
@@ -322,4 +322,56 @@ public function css_and_js() {
 		<?php
 	}
 
+	public function register_reject_feedback_js( $template, $translation_set ) {
+
+		if ( 'translations' !== $template ) {
+			return;
+		}
+
+		wp_register_script( 'gp-reject-feedback-js', plugins_url( '/../js/reject-feedback.js', __FILE__ ), array( 'jquery', 'gp-common', 'gp-editor' ), '0.0.1' );
```

Comment:
I've added script dependencies here so that it will be loaded at the right place in GlotPress core.